features/other_b/skin.feature

```diff
@@ -300,6 +302,7 @@ Feature: creating and editing skins
       And I fill in "CSS" with ".myclass { -moz-box-sizing: border-box; -webkit-transition: opacity 2s; }"
       And I submit
     Then I should see "Skin was successfully created"
+    Then the cache of the skin on "skin with prefixed property" should expire after I save the skin
 
   Scenario: #workskin selector prefixing
     Given basic skins
```

Comment:
Actually It will expire on any site skin being changed as that was done to make the cache key calculation simple as invalidating the cache has not got a high cost and making the code complex is costly in getting the code right and means we are more likely to make errors and the cache key code has to run all the time so keeping it as simple as possible is good.